pydm/widgets/baseplot.py

```diff
@@ -610,7 +608,11 @@ def auto_range(self) -> bool:
         -------
         bool
         """
-        return self._auto_range
+        if self.orientation == "left" or self.orientation == "right":
+            axis = ViewBox.YAxis
+        elif self.orientation == "top" or self.orientation == "bottom":
+            axis = ViewBox.XAxis
+        return self.linkedView().autoRangeEnabled()[axis]  # ViewBox axes map to 0 and 1
```

Comment:
Will probably want to cast to a bool since pyqtgraph returns a float instead of True. (Shows up as a 1.0 in designer)

`            'autoRange': [True, True],  ## False if auto range is disabled,
                                        ## otherwise float gives the fraction of data that is visible`